### ISSUE TITLE
Must Python scripts write in SL4A format?

### ISSUE THREAD

**Issue by twlkyao** [devpack/android-python27#2]
I have look throgh all the wiki of android-python27 on code google site, must we write Python scripts with by using SL4A? Can we just write Python which get some data and pass to Android Activity for future use?

**Comment by michaelrinderle** [comment]
Yes and no. You still need sl4a to pass data to another activity, i.e. AndroidFacade.
